contracts/vesting/src/vesting/mod.rs

```diff
@@ -1,19 +1,21 @@
-use crate::contract::VESTING_PERIOD;
 use cosmwasm_std::{Timestamp, Uint128};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 mod account;
 pub use account::*;
 
+use crate::messages::VestingSpecification;
+
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct VestingPeriod {
     pub start_time: u64,
+    pub period_seconds: u32,
 }
 
 impl VestingPeriod {
     pub fn end_time(&self) -> Timestamp {
-        Timestamp::from_seconds(self.start_time + VESTING_PERIOD)
+        Timestamp::from_seconds(self.start_time + self.period_seconds as u64)
```

Comment:
if you have to cast it here to u64, why not keep `period_seconds` as u64 to begin with?

Reply:
Yeah, probably best, I always try to save on types, which usually turns out to be silly